packages/parser/src/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { type ZodSchema, z } from 'zod';
+
+const JSONStringified = (schema: ZodSchema) =>
```

Comment:
Can we add a docstring that describes how to use it with an example?

Reply:
oh absolutely, I was in a rush to ship 😄